### PR TITLE
Remove django create account and login form, remove csrf

### PR DESCRIPTION
**Pull request by ProgrammingJoe**

### CODE REVIEW

File: meteor/views.py [hunk 1]
```diff
@@ -31,7 +30,7 @@ def authenticateuser(request):
 
     email, password = (request.POST[k] for k in ('email', 'password'))
```
Comment (ProgrammingJoe) [inline meteor/views.py]: @rankinstein Login request

File: meteor/views.py [hunk 2]
```diff
@@ -31,7 +30,7 @@ def authenticateuser(request):
...
     user = authenticate(request, email=email, password=password)
-    
+
     if not user:
         return JsonResponse({'error': 'user could not be found for that email + password combination'})
```
Comment (rankinstein) [inline meteor/views.py]: The response from sign-in should be a json with the email, organization, account permission level, and any other info that is pertinent to the front end.

File: meteor/views.py [hunk 3]
```diff
@@ -59,10 +58,10 @@ def create_user(data):
...
         return JsonResponse({'error': 'invalid email ' + str(e)})
-    
+
     if User.objects.filter(email=email):
         return JsonResponse({'error': 'User already exists for that email'})
-        
+
     new = User(first_name=first, last_name=last, organization=organization, email=email)
     new.set_password(password)
     new.save()
```
Comment (rankinstein) [inline meteor/views.py]: Create account should respond with a JSON.  Ill probably redirect them to the sign-up view so no need to return user info.

**Comment by rankinstein** [comment]
For the JSON responses to clean up the front end handling include some status info
`
{
success: true/false,
// other data
}
`

for error cases use a short string such 'EMAIL_ALREADY_EXISTS' and the front-end can handle all messages (such as in the future, internationalization).

error: 'EMAIL_ALREADY_EXISTS',
message: 'There is already a user with this email'

**Comment by kradical** [comment]
@rankinstein is it fair to do different response codes instead of success: true/false? we could start out with just 400s for any error and specialize them as we need to. I agree with the SHORT_STRING that the frontend can internationalize.

**Comment by rankinstein** [comment]
@kradical good point. response status would be a good approach

File: meteor/views.py (hunk 3 above)
Comment (rankinstein) [inline meteor/views.py]: So following up on what @kradical said. if the account creation is successful just returning a 200 status should suffice. Then ill redirect to a sign-in form.  I guess in the future will want something less exploitable such as email verification etc but that is outside what we need to do for the prototype.

**Comment by kradical** [comment]
Merged. Joe or I will make the changes.

**Pull request merged**